src/components/OpenGameCard.tsx

```diff
@@ -33,7 +33,7 @@ export class OpenGameEntry extends React.PureComponent<Props, State> {
 
     const stake = openGame.stake;
     // TODO: should not hardcode the number of rounds
-    const buyin = stake.mul(new BN(5));
+    const buyin = new BN(5000000000000000);
```

Comment:
@tomclose Does the buy-in depend on the wager per round?

Reply:
This is a temporary fix to work around `BN` problems. Will be dealing with this next.